src/odemis/gui/util/test/img_test.py

```diff
@@ -1109,6 +1190,49 @@ def test_first(self):
         for s, md in zip(self.streams, orig_md):
             self.assertEqual(md, s.raw[0][0].metadata)
 
+        # Checking the ratio between height/width of each stream
+        for data_gray, ratio in zip(exp_data_gray, self.image_ratio):
+            new_image_ratio = data_gray.shape[0] / data_gray.shape[1]
+            self.assertLess(abs(new_image_ratio - ratio) / ratio, 0.3)
+
+    def test_narrow_rectangular(self):
+        '''
+        Test printing an stream which has an high resolution an is an extremely narrow rectangular image
+        '''
+        self.setUp_custom((1000, 10000), (1000, 10000))
+        time.sleep(2.5)  # Extra waiting time cause loading of large picture takes longer
+        orig_md = [s.raw[0][0].metadata.copy() for s in self.streams]
+
+        # Print ready format
+        view_hfw = (8.191282393266523e-03/3, 6.205915392651362e-04/3)
+        view_pos = [-0.001203511795256, -0.000295338300158]
+
+        draw_merge_ratio = 0.3
+        self.streams[1]._updateImage()
```

Comment:
You really shouldn't need to call this function. Is that still needed? If so, why?